src/ca/legacy/pcas/generic/pvExistReturn.cc

```diff
@@ -30,13 +34,15 @@ const pvExistReturn & pvExistReturn::operator = ( pvExistReturnEnum rhs )
 {
 	this->status = rhs;
 	this->address.clear ();
+   this->minorProtocolVersion= 0;
```

Comment:
Please use consistent indentation in this file.

Reply:
Will fix.